### ISSUE TITLE
[Optimization Plan] Change Validation Steps

### ISSUE THREAD

**Issue by softcat477** [DDMAL/Paco_classifier#10]
### Goal: Change validation steps from `100` to something else
1. `validation_steps` is fixed to `100` for an unknown reason. 100 is way larger than the number of training steps, which is `32` using the best training parameters.
2. Reduce validation steps to a number that does not result in a jittering validation accuracy curve.
### Groceries
1. The best training param is `#samples_per_epoch=1000`, `#epoch=50`,`#batch_size=32`. So the steps per epoch in training set is `#samples_per_epoch/#batch_size=31.25~=32`.
